src/form-field/form-field.svelte

```diff
@@ -0,0 +1,49 @@
+<script>
+  import classes from '../_utils/classes.js';
+
+  let _class = null;
+  export { _class as class };
+  export let nameClass = null;
+  export let helpClass = null;
+
+  export let name = null;
+  export let help = null;
+
+  export let required = false;
+  export let optional = false;
+  export let errors = null;
+</script>
+
+<div class={classes('form-field', _class)}>
+  <slot name="description">
+    <div class="description">
+      {#if name != null}
+        <div class={classes('name', nameClass)}>
+          {name}
+          {#if required}
+            <span class="required">*</span>
+          {/if}
+        </div>
+      {/if}
+      {#if help != null}
+        <div class={classes('help', helpClass)}>{help}</div>
+      {/if}
+    </div>
+  </slot>
+  <div class="field">
+    <slot />
+    {#if errors != null}
+      {#each errors as error}
+        {#if error}<div class="message error">{error}</div>{/if}
```

Comment:
Maybe also a slot with the error as a parameter?

Reply:
In this case it would make matters only less clear, since the default slot is occupied. With the way of specifying errors as in the demo, this way ends up pretty clean.